src/engine/Settings.cs

```diff
@@ -339,6 +343,19 @@ public void ApplyWindowSettings()
         OS.VsyncEnabled = VSync;
     }
 
+    /// <summary>
+    ///   Applies current language settings to any applicable engine systems.
+    /// </summary>
+    public void ApplyLanguageSettings()
+    {
+        TranslationServer.SetLocale(SelectedLanguage);
```

Comment:
Shouldn't this function skip doing anything if `SelectedLanguage` is null?

Reply:
If the locale does not exist, Godot will just fall back to the first one (en). I can add an if() if you prefer.

Reply:
If the C# new Locale also works with null to select the default locale, then this is actually perfect without a null check.